Resources/input/fsh/examples/Example_Parameters_Create.fsh

```diff
@@ -0,0 +1,7 @@
+Instance: OperationCreateParametersInputExample
+InstanceOf: GEM_ERP_PR_CreateOperation_Input
+Title: "$create Operation Parameters sample"
+Usage: #example
+* parameter[workflowType].name = "workflowType"
+* parameter[workflowType].valueCoding.system = "https://gematik.de/fhir/erp/CodeSystem/GEM_ERP_CS_FlowType"
```

Comment:
Könnte man vereinfachen in:
* parameter[workflowType].valueCoding = GEM_ERP_CS_FlowType#160